mpitest/test_sqlite.py

```diff
@@ -329,7 +329,7 @@ def test_driver_doesnt_records_metadata(self):
         self.recorder.options['record_metadata'] = False
         prob.setup(check=False)
 
-        self.recorder.close()
+        prob.cleanup()
 
         self.assertMetadataRecorded(None)
 
```

Comment:
Note: I added two tests two most of the recorder tests, so they will have to replace close with cleanup too.